recipe/README.md

```diff
@@ -170,15 +194,14 @@ then:
 
 > [!WARNING]
 > The condition `timestamp_lt` is required to prevent your patch from modifying
-> any packages built in the future. Don't forget to calculate it with `python -c
-> "import time; print(f'{time.time():.0f}000')"` and include it in the `if:`
```

Comment:
Why hide the Python code from the user? Why not just append something like "or run `pixi run timestamp`"?